src/table/settings/linkgraph_settings.ini

```diff
@@ -37,26 +37,26 @@ startup  = false
 var      = linkgraph.recalc_interval
 type     = SLE_UINT16
 from     = SLV_183
-def      = 4
-min      = 2
-max      = 32
-interval = 2
-str      = STR_CONFIG_SETTING_LINKGRAPH_INTERVAL
+def      = 8
+min      = 4
+max      = 64
+interval = 4
+str      = STR_CONFIG_SETTING_LINKGRAPH_RECALC_INTERVAL
 strval   = STR_JUST_COMMA
-strhelp  = STR_CONFIG_SETTING_LINKGRAPH_INTERVAL_HELPTEXT
+strhelp  = STR_CONFIG_SETTING_LINKGRAPH_RECALC_INTERVAL_HELPTEXT
 extra    = offsetof(LinkGraphSettings, recalc_interval)
 
 [SDT_VAR]
 var      = linkgraph.recalc_time
 type     = SLE_UINT16
 from     = SLV_183
-def      = 16
+def      = 32
 min      = 1
-max      = 4096
+max      = 8192
```

Comment:
I just don't even .. what a large number :P But okay .. please also max this something more sane, like `10000` or something.

Reply:
It's now 9,000, which is 150 minutes or 100 of the maximum linkgraph recalc intervals. Hope that's even enough for you. 😉